pkg/testutil/testregistry/testregistry_linux.go

```diff
@@ -468,3 +478,12 @@ func NewWithBasicAuth(base *testutil.Base, user, pass string, port int, tls bool
 	}
 	return NewRegistry(base, ca, port, auth, nil)
 }
+
+func SafeRandomString(n int) string {
+	b := make([]byte, n)
+	_, _ = rand.Read(b)
+	// XXX WARNING there is something in the registry (or more likely in the way we generate htpasswd files)
+	// that is broken and does not resist truly random strings
+	// return string(b)
```

Comment:
This comment should be removed?

Reply:
You mean, just line 487, right?
Will do.